src/evidently/ui/errors.py

```diff
@@ -1,17 +1,6 @@
-import json
-
 from litestar import Response
 
 
-class JSONResponse(Response):
```

Comment:
This isn't needed; The `Response` class can already deal with everything Litestar can serialise.